mods/ra/weapons/explosions.yaml

```diff
@@ -142,15 +142,41 @@ CivPanicExplosion:
 		Delay: 1
 
 BarrelExplode:
+	Warhead@Cluster: FireCluster
+		Weapon: BarrelCluster
+		Dimensions: 3,3
+		Footprint: _X_ X_X _X_
+	Warhead@1Dam: SpreadDamage
+		Spread: 325
+		Falloff: 1000, 368, 135, 50, 18, 7, 0
+		Delay: 5
+		ValidTargets: GroundActor, Trees
+		Versus:
+			None: 120
+			Light: 50
+			Wood: 20
+			Concrete: 10
+	Warhead@2Eff: CreateEffect
+		Explosions: building
+		ImpactSounds: kaboom25.aud
+		Delay: 5
+	Warhead@Smu: LeaveSmudge
+		SmudgeType: Crater
+		Delay: 1
+
+BarrelCluster:
 	Inherits: ^Explosion
+	Projectile: Bullet
```

Comment:
In my opinion `InstantHit` would make more sense, untested though.

Reply:
The idea is to have a swelling effect with the explosions. Our barrels chain-explode a bit faster than the OG at the moment.

Reply:
Wouldn't a higher delay give the same effect, though? Or am I missing something?

Reply:
Tested your way and that'll work.